hack/local-up-cluster.sh

```diff
@@ -897,6 +918,8 @@ if [[ "${START_MODE}" != "kubeletonly" ]]; then
   start_apiserver
```

Comment:
start_apiserver already waits for healthz... waiting for rbac should not be necessary